miio/huizuo.py

```diff
@@ -0,0 +1,137 @@
+import logging
+import click
+
+from .click_common import command, format_output
+from .device import Device
+from .exceptions import DeviceException
+
+"""
+Basic implementation for HUAYI HUIZUO PISCES For Bedroom (huayi.light.pis123) lamp
+
+This lamp is white color only and supports dimming and control of the temperature from 3000K to 6400K
+Specs: https://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:light:0000A001:huayi-pis123:1
+
+"""
```

Comment:
This should be moved to the top of the file (that's the place for module level docstrings).

Reply:
Moved